bundles/automation/org.eclipse.smarthome.automation.module.script.extensions.rulesupport/ESH-INF/automation/moduletypes/PrivateScriptedTypes.json

```diff
@@ -0,0 +1,41 @@
+{  
```

Comment:
The implementation of these private type functions are bound be the ScriptEngine, they cannot simply be serialized and later on evaluated by ScriptAction, ScriptCondition etc.. Here I save the function handler in combination with a "privId".